<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Regional Analytics Tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pictures.qrc">
    <normaloff>:/cat.png</normaloff>:/cat.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="nameFilelabel">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>460</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(208, 188, 130);</string>
    </property>
    <property name="text">
     <string>Path</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="nameRegion">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="nameRegionlabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>321</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(208, 188, 130);</string>
    </property>
    <property name="text">
     <string>Enter the name of the region:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="selectColumn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>300</y>
      <width>281</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>Natural population growth</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Birth rate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Death rate</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>General demographic weight</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Urbanization</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="nameColumnlabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>300</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(208, 188, 130);</string>
    </property>
    <property name="text">
     <string>Select a column:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="calculateMetricsButton">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>370</y>
      <width>221</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Calculate and draw</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorLabel">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>440</y>
      <width>411</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(255, 0, 0)</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTableWidget" name="table">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>1131</width>
      <height>431</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>440</y>
      <width>400</width>
      <height>100</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(208, 188, 130);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableMetrics">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>450</y>
      <width>191</width>
      <height>181</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>108</number>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <row>
     <property name="text">
      <string>min</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>max</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>medium</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Merics:</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="1" column="0">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
    <item row="2" column="0">
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="openFileButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Open File</string>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/back.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="graph">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>560</y>
      <width>600</width>
      <height>320</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>tableMetrics</zorder>
   <zorder>nameFilelabel</zorder>
   <zorder>nameRegion</zorder>
   <zorder>nameRegionlabel</zorder>
   <zorder>selectColumn</zorder>
   <zorder>nameColumnlabel</zorder>
   <zorder>calculateMetricsButton</zorder>
   <zorder>errorLabel</zorder>
   <zorder>table</zorder>
   <zorder>infoLabel</zorder>
   <zorder>openFileButton</zorder>
   <zorder>graph</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="pictures.qrc"/>
 </resources>
 <connections/>
</ui>
